_docs-2/configuration/files.md

```diff
@@ -38,16 +38,16 @@ Accumulo processes. See the [quick start] for help with configuring this file.
 
 ### log4j-service.properties
```

Comment:
The names of these headers no longer match the file names in the config directory. They should probably be updated as well.

Reply:
Sorry, missed that, now fixed.